src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterReceiver.cs

```diff
@@ -224,30 +243,54 @@ public Task MessagesDeliveredAsync(IList<IBatchContainer> messages)
+#else
+        private static async Task<EventHubReceiver> CreateReceiver(EventHubPartitionSettings partitionSettings, string offset, Logger logger) 
+#endif
         {
             bool offsetInclusive = true;
-            EventHubClient client = EventHubClient.CreateFromConnectionString(partitionSettings.Hub.ConnectionString, partitionSettings.Hub.Path);
+#if NETSTANDARD
+            var connectionStringBuilder = new EventHubsConnectionStringBuilder(partitionSettings.Hub.ConnectionString)
+            {
+                EntityPath = partitionSettings.Hub.Path
+            };
+            EventHubClient client = EventHubClient.CreateFromConnectionString(connectionStringBuilder.ToString());
+#else
+            EventHubClient client = EventHubClient.CreateFromConnectionString(partitionSettings.Hub.ConnectionString, partitionSettings.Hub.Path); 
+
             EventHubConsumerGroup consumerGroup = client.GetConsumerGroup(partitionSettings.Hub.ConsumerGroup);
             if (partitionSettings.Hub.PrefetchCount.HasValue)
             {
                 consumerGroup.PrefetchCount = partitionSettings.Hub.PrefetchCount.Value;
             }
+#endif
             // if we have a starting offset or if we're not configured to start reading from utc now, read from offset
-            if (!partitionSettings.Hub.StartFromNow || offset != EventHubConsumerGroup.StartOfStream)
+            if (!partitionSettings.Hub.StartFromNow || 
+                offset != EventHubConstants.StartOfStream)
             {
                 logger.Info("Starting to read from EventHub partition {0}-{1} at offset {2}", partitionSettings.Hub.Path, partitionSettings.Partition, offset);
             }
             else
             {
                 // to start reading from most recent data, we get the latest offset from the partition.
-                PartitionRuntimeInformation patitionInfo =
+#if NETSTANDARD
+                EventHubPartitionRuntimeInformation partitionInfo =
+#else
+                PartitionRuntimeInformation partitionInfo = 
+#endif
                     await client.GetPartitionRuntimeInformationAsync(partitionSettings.Partition);
-                offset = patitionInfo.LastEnqueuedOffset;
+                offset = partitionInfo.LastEnqueuedOffset;
                 offsetInclusive = false;
                 logger.Info("Starting to read latest messages from EventHub partition {0}-{1} at offset {2}", partitionSettings.Hub.Path, partitionSettings.Partition, offset);
             }
-            return await consumerGroup.CreateReceiverAsync(partitionSettings.Partition, offset, offsetInclusive);
+#if NETSTANDARD
+            PartitionReceiver receiver = client.CreateReceiver(partitionSettings.Hub.ConsumerGroup, partitionSettings.Partition, offset, offsetInclusive);
+            receiver.PrefetchCount = partitionSettings.Hub.PrefetchCount.Value;
```

Comment:
watch out, as in the previous code, PrefetchCount might have been null. Look at how it was before:
```
	 if (partitionSettings.Hub.PrefetchCount.HasValue)
              {	
                  consumerGroup.PrefetchCount = partitionSettings.Hub.PrefetchCount.Value;		                  consumerGroup.PrefetchCount = partitionSettings.Hub.PrefetchCount.Value;
              }	
```